gtk2_ardour/add_route_dialog.cc

```diff
@@ -598,11 +591,31 @@ AddRouteDialog::type_wanted()
 	} else if (str == _("Foldback Busses")) {
 		return FoldbackBus;
 	} else {
-		assert (0);
```

Comment:
I ran into problems when selecting a non-builtin-type and trying to save/load it. I defaulted to AudioTrack, but maybe this is not what we want.